packages/connect/protocol-connect.js

```diff
@@ -12,5 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// workaround for bundlers that do not support subpath exports.
-export * from "./dist/esm/protocol-connect/index.js";
+/* eslint-disable */
```

Comment:
Instead of disabling eslint completely, please use `eslint-disable-line` or `eslint-disable-next-line`.

Reply:
@timostamm there are multiple files in `connect-web-test` that also use this directive (`packages/connect-web-test/karma-fixup-queueMicrotask.js` for example). should I change those also?

Reply:
Unless it turns out to need a linter disable on every second line or so, yes.